### PR TITLE
Quality improve

### PR DESCRIPTION
**Pull request by piotr921**
Remove unused imports, bugs found by Intellij and opening trello board in quality script.

### CODE REVIEW

File: checkQuality.sh [hunk 1]
```diff
@@ -2,13 +2,10 @@
...
-firefox https://trello.com/b/ONK9r1ZU/battleships &
-
 #create quality analysis
 mvn clean install
 mvn site
 mvn sonar:sonar
 
 #display results
-firefox -new-tab http://localhost:9000 -new-tab target/site/index.html &
+firefox http://localhost:9000 target/site/index.html &
```
Comment (mprtcz) [inline checkQuality.sh]: 👍

File: src/main/java/com/spanish_inquisition/battleship/client/game/FleetInitializer.java [hunk 2]
```diff
@@ -24,7 +25,7 @@ public FleetInitializer(BoardController boardController) {
     public void setUpShips() {
         boardTiles = boardController.getBoardsIndexTiles();
         shipPlaces = Fleet.getRandomFleet().getShipPlaces();
-        shipPlaces.forEach(this::changeTileStyle);
+        Platform.runLater(() -> shipPlaces.forEach(this::changeTileStyle));
```
Comment (mprtcz) [inline src/main/java/com/spanish_inquisition/battleship/client/game/FleetInitializer.java]: 👍

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java [hunk 3]
```diff
@@ -68,17 +65,17 @@ private void assignSocketAndNameTo(String nameString) {
     @Test
     public void testIfFleetsAreAssigned() {
         GameState pss = new PlacingShipsState(players, messageBus);
-        messageBus.addMessage(1,0, Header.FLEET_REQUEST.name()+":[0,2,4,6,8,9,20,21,23,24,26,27,28,40,41,42,44,45,46,47];");
-        messageBus.addMessage(2,0, Header.FLEET_REQUEST.name()+":[0,2,4,6,8,9,20,21,23,24,26,27,28,40,41,42,44,45,46,47];");
+        messageBus.addMessage(1, 0, Header.FLEET_REQUEST.name() + ":[0,2,4,6,8,9,20,21,23,24,26,27,28,40,41,42,44,45,46,47];");
```
Comment (mprtcz) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: 👍

File: src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java [hunk 4]
```diff
@@ -15,25 +15,24 @@
...
 
-
 /**
  * @author Michal_Partacz
  * This class will make sure the board of buttons is properly initialized and with every event handlers needed.
  */
 public class GameBoardBuilder {
     public static final int BOARD_SIZE_WITH_LABELS = 11;
-    public static final double FIELD_SIZE = 30;
+    private static final double FIELD_SIZE = 30;
```
Comment (Ice94) [inline src/main/java/com/spanish_inquisition/battleship/client/board/GameBoardBuilder.java]: :1st_place_medal:

File: src/test/java/com/spanish_inquisition/battleship/client/board/BoardTileTest.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    public void setupJavaFx() {
+        final CountDownLatch latch = new CountDownLatch(1);
+        SwingUtilities.invokeLater(() -> {
+            new JFXPanel(); // initializes JavaFX environment
+            latch.countDown();
+        });
+    }
+
+    @Test
+    public void shouldSetLabels() {
```
Comment (Ice94) [inline src/test/java/com/spanish_inquisition/battleship/client/board/BoardTileTest.java]: Perfect name! :1st_place_medal:

File: src/test/java/com/spanish_inquisition/battleship/client/board/GameBoardTest.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    public void setupJavaFx() {
+        final CountDownLatch latch = new CountDownLatch(1);
+        SwingUtilities.invokeLater(() -> {
+            new JFXPanel(); // initializes JavaFX environment
+            latch.countDown();
+        });
+    }
+
+    @Test
+    public void shouldAddTileToHashMap() {
```
Comment (Ice94) [inline src/test/java/com/spanish_inquisition/battleship/client/board/GameBoardTest.java]: Nice name.

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java [hunk 7]
```diff
@@ -42,7 +39,7 @@ public void initialize() {
         new Thread(() -> assignSocketAndNameTo("Name 1")).run();
         new Thread(() -> assignSocketAndNameTo("Name 2")).run();
         players = new Players();
-        for(int i = 1; i < 3; i++) {
+        for (int i = 1; i < 3; i++) {
```
Comment (Ice94) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: magic numbers :-1:

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java (hunk 3 above)
Comment (Ice94) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: Magic numbers!
Use data provider.

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java [hunk 8]
```diff
@@ -68,17 +65,17 @@ private void assignSocketAndNameTo(String nameString) {
...
-        messageBus.addMessage(1,0, Header.FLEET_REQUEST.name()+":[0,2,4,6,8,9,20,21,23,24,26,27,28,40,41,42,44,45,46,47];");
-        messageBus.addMessage(2,0, Header.FLEET_REQUEST.name()+":[0,2,4,6,8,9,20,21,23,24,26,27,28,40,41,42,44,45,46,47];");
+        messageBus.addMessage(1, 0, Header.FLEET_REQUEST.name() + ":[0,2,4,6,8,9,20,21,23,24,26,27,28,40,41,42,44,45,46,47];");
+        messageBus.addMessage(2, 0, Header.FLEET_REQUEST.name() + ":[0,2,4,6,8,9,20,21,23,24,26,27,28,40,41,42,44,45,46,47];");
         pss.transform();
 
         List<List<Ship>> playersShips = new ArrayList<>();
         List<Player> playersList = players.getBothPlayers();
-        for(int i = 0; i < 2; i++) {
+        for (int i = 0; i < 2; i++) {
```
Comment (Ice94) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: ```private final int NUMBER_OF_PLAYERS = 2```

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java [hunk 9]
```diff
@@ -68,17 +65,17 @@ private void assignSocketAndNameTo(String nameString) {
...
             playersShips.add(playersList.get(i).getFleet().getShips());
         }
 
-        for(List<Ship> playerShips : playersShips) {
+        for (List<Ship> playerShips : playersShips) {
             SoftAssert soft = new SoftAssert();
             soft.assertEquals(10, playerShips.size());
             soft.assertEquals(1, playerShips.get(0).getShipPoints().size());
```
Comment (Ice94) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: For example:

```private final int ONE_MAST_SHIP;```
```private final int FIRST_SHIP;```

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/ShotStateTest.java [hunk 10]
```diff
@@ -41,7 +37,7 @@ public void initialize() {
         new Thread(() -> assignSocketAndNameTo("Name 1")).run();
         new Thread(() -> assignSocketAndNameTo("Name 2")).run();
         players = new Players();
-        for(int i = 1; i < 3; i++) {
+        for (int i = 1; i < 3; i++) {
```
Comment (Ice94) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/ShotStateTest.java]: Also magic number.

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java (hunk 7 above)
Comment (piotr921) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: done

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java (hunk 3 above)
Comment (piotr921) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: Added data provider

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java (hunk 8 above)
Comment (piotr921) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: done

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java (hunk 9 above)
Comment (piotr921) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/PlacingShipsStateTest.java]: done

File: src/test/java/com/spanish_inquisition/battleship/server/game_states/ShotStateTest.java (hunk 10 above)
Comment (piotr921) [inline src/test/java/com/spanish_inquisition/battleship/server/game_states/ShotStateTest.java]: done

File: src/main/java/com/spanish_inquisition/battleship/server/game_states/GameState.java [hunk 11]
```diff
@@ -1,11 +1,8 @@
...
 
 import com.spanish_inquisition.battleship.server.MessageBus;
-import com.spanish_inquisition.battleship.server.Player;
 import com.spanish_inquisition.battleship.server.Players;
 
-import java.util.List;
-
 public abstract class GameState {
     protected Players players;
     protected MessageBus requestBus;
```
Comment (Tymoteuszauke) [inline src/main/java/com/spanish_inquisition/battleship/server/game_states/GameState.java]: Dont name a field requestSomething it's a bit misleading (reminds me method name)

File: src/main/java/com/spanish_inquisition/battleship/server/game_states/ShotState.java [hunk 12]
```diff
@@ -5,8 +5,6 @@
 import com.spanish_inquisition.battleship.server.Player;
 import com.spanish_inquisition.battleship.server.Players;
 
-import java.util.List;
```
Comment (Tymoteuszauke) [inline src/main/java/com/spanish_inquisition/battleship/server/game_states/ShotState.java]: :+1:

File: src/main/java/com/spanish_inquisition/battleship/server/game_states/PlayerActionState.java [hunk 13]
```diff
@@ -2,11 +2,8 @@
 
 import com.spanish_inquisition.battleship.common.Header;
 import com.spanish_inquisition.battleship.server.MessageBus;
-import com.spanish_inquisition.battleship.server.Player;
 import com.spanish_inquisition.battleship.server.Players;
 
-import java.util.List;
-
```
Comment (Tymoteuszauke) [inline src/main/java/com/spanish_inquisition/battleship/server/game_states/PlayerActionState.java]: :+1:

**Pull request merged**